src/data/gqlQueries.js

```diff
@@ -72,14 +72,17 @@ export async function createRequest(request) {
 
 export async function updateRequest(request) {
   // TODO: When API is updated to erase values when we send `null`, update `|| something` to `|| null`
+
+  const neededBefore = request.needed_before && request.needed_before.split("T")[0] || null
```

Comment:
same as our conversation in src/components/CreateOrUpdateRequest.svelte but not necessary.